components/country/apps.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+
+import SectionHeader from './section-header'
+import { SimpleBox } from './box'
+import PeriodFilter from './period-filter'
```

Comment:
I guess this is an unused import. Do we have linting in place to detect unused imports?

Reply:
Yes. The linting is in place. Most of these imports have been in constant flux while developing. I am supposed to do a full pass to fix such linting issues.

Reply:
This particular import is temporarily unused. We should bring back PeriodFilter component back in the app in the near future. Resolving for now because I removed all the other unused imports.